extensions-contrib/opentelemetry-extensions/src/main/java/org/apache/druid/data/input/opentelemetry/protobuf/traces/OpenTelemetryTracesProtobufConfiguration.java

```diff
@@ -0,0 +1,300 @@
+  {
+    return endTimeDimension;
+  }
+
+  @JsonProperty
+  public String getStatusCodeDimension()
+  {
+    return statusCodeDimension;
+  }
+
+  @JsonProperty
+  public String getStatusMessageDimension()
+  {
+    return statusMessageDimension;
+  }
+
+  @JsonProperty
+  public String getKindDimension()
+  {
+    return kindDimension;
+  }
+
+  @JsonProperty
+  public String getSpanAttributePrefix()
+  {
+    return spanAttributePrefix;
+  }
+
+  private OpenTelemetryTracesProtobufConfiguration(
+      Builder builder
+  )
+  {
+    this.resourceAttributePrefix = builder.resourceAttributePrefix;
+    this.spanAttributePrefix = builder.spanAttributePrefix;
+    this.nameDimension = builder.nameDimension;
+    this.spanIdDimension = builder.spanIdDimension;
+    this.parentSpanIdDimension = builder.parentSpanIdDimension;
+    this.traceIdDimension = builder.traceIdDimension;
+    this.endTimeDimension = builder.endTimeDimension;
+    this.statusCodeDimension = builder.statusCodeDimension;
+    this.statusMessageDimension = builder.statusMessageDimension;
+    this.kindDimension = builder.kindDimension;
+  }
+
+  public static Builder newBuilder()
+  {
+    return new Builder();
+  }
+
+  @Override
+  public boolean equals(Object o)
+  {
+    if (this == o) {
+      return true;
+    }
+    if (o == null || getClass() != o.getClass()) {
+      return false;
+    }
+    OpenTelemetryTracesProtobufConfiguration that = (OpenTelemetryTracesProtobufConfiguration) o;
+    return Objects.equals(spanAttributePrefix, that.spanAttributePrefix)
+           && Objects.equals(nameDimension, that.nameDimension)
+           && Objects.equals(spanIdDimension, that.spanIdDimension)
+           && Objects.equals(parentSpanIdDimension, that.parentSpanIdDimension)
+           && Objects.equals(traceIdDimension, that.traceIdDimension)
+           && Objects.equals(endTimeDimension, that.endTimeDimension)
+           && Objects.equals(statusCodeDimension, that.statusCodeDimension)
+           && Objects.equals(statusMessageDimension, that.statusMessageDimension)
+           && Objects.equals(kindDimension, that.kindDimension)
+           && Objects.equals(resourceAttributePrefix, that.resourceAttributePrefix);
+  }
+
+  @Override
+  public int hashCode()
+  {
+    return Objects.hash(
+        spanAttributePrefix,
+        nameDimension,
+        spanIdDimension,
+        parentSpanIdDimension,
+        traceIdDimension,
+        endTimeDimension,
+        statusCodeDimension,
+        statusMessageDimension,
+        kindDimension,
+        resourceAttributePrefix
+    );
+  }
+
+  @JsonPOJOBuilder(withPrefix = "set")
+  public static class Builder
+  {
+    @JsonProperty("spanAttributePrefix")
+    private String spanAttributePrefix = "span.";
+
+    @JsonProperty("resourceAttributePrefix")
+    private String resourceAttributePrefix = "resource.";
+
+    @JsonProperty("nameDimension")
+    private String nameDimension = "name";
```

Comment:
I'm not necessarily asking to change it. I think we want it to be somewhat consistent with our metric converter.
For the metrics converter, we initially used `name` for the for the metric name field, but then we switched it to `metric` (it was unclear what it meant to work with a column called `name` when writing sql queries). So maybe `span` would be more consistent here with what we do for metrics.

It might also make sense to reserve the `.` separator for things that are actually nested sub-fields in the span, and not just individual fields to avoid collisions, so maybe we should keep `span_id`, which aligns with what the field is already called in the proto definition https://github.com/open-telemetry/opentelemetry-proto/blob/main/opentelemetry/proto/trace/v1/trace.proto#L95

Reply:
done. Changed to `span_name` and removed `.` separator for non-nested fields.